docs/examples/suggesthub/app/main.py

```
@@ -0,0 +1,456 @@
+@app.post("/api/bob/message")
+async def bob_message(payload: BobMessageRequest) -> dict[str, Any]:
+    fallback = deterministic_intake(payload.message, payload.thread_id)
+    try:
+        result = await get_bob_agent().invoke(
+            payload.message,
+            thread_id=payload.thread_id,
+        )
+        answer = getattr(result, "answer", None) or str(result)
+        if getattr(result, "error", None):
+            answer = fallback["reply"]
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)